docker-compose.developer.yml

```diff
@@ -6,6 +6,13 @@ services:
             - 8080:80
         networks:
             - network
+    serverroom-api:
```

Comment:
Nie ma pliku dockerowego, który by był budowany i możliwy do wczytania. Daj znać, jeśli będziesz z tym pomocy potrzebował + stworzę budowanie automatyczne i publikowanie do rejstru obrazów jak będzie to gotowe